### ISSUE TITLE
Nexa node mining issue

### ISSUE THREAD

**Issue by Ruinous101** [bzminer/bzminer#205]
Hi, 

On PC windowX with 3090 trying to mine local nexa qt node with no success. Node is sync'd and bzminer nexa batch file is below. Not sure were to start troubleshooting. Using 127.0.0.1 gives Nexa node authentication error, using Local as seen below starts miner but then gives watchdog reset/restart message over and over (no mining ever starts). Only setting in the node conf file is "server=1"

bzminer -a nexa -w <mynexaaddress> -p node+tcp://Local:7227 --nc 1

basically looking to test self mining setup to use at other sites.
